frontend/src/pages/modelServing/ModelServingRoutes.tsx

```diff
@@ -3,15 +3,27 @@ import { Navigate, Routes, Route } from 'react-router-dom';
 import ModelServingContextProvider from './ModelServingContext';
 import ModelServingMetricsWrapper from './screens/metrics/ModelServingMetricsWrapper';
 import ModelServingGlobal from './screens/global/ModelServingGlobal';
+import { isModelMetricsEnabled } from './screens/metrics/utils';
+import { useDashboardNamespace } from 'redux/selectors';
+import { useAppContext } from 'app/AppContext';
 
 const ModelServingRoutes: React.FC = () => {
+  const { dashboardNamespace } = useDashboardNamespace();
+  const { dashboardConfig } = useAppContext();
```

Comment:
I'm a little disappointed you didn't make the utility a hook and hide these away inside it... we should always strive to avoid extraneous dependencies like this from leaking.

Reply:
Yeah, i was so focused on getting it done that i didn't consider it, i can get it done in a minute!

Reply:
Fixed, added as a separate commit, I'm sorry I didn't get it from the beginning.